.github/workflows/release.yml

```diff
@@ -27,15 +27,15 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y autoconf automake build-essential cmake gcc-11 gettext libtool-bin locales ninja-build pkg-config unzip
+          sudo apt-get install -y autoconf automake build-essential cmake gettext libtool-bin locales ninja-build pkg-config unzip
```

Comment:
Sounds we explicitly install gcc-10 here, if it's not the default?  Not sure what's currently pulling it in, but wouldn't want to rely on that.

Reply:
No, we're not. [ubuntu-20.04](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#language-and-runtime) ships with `gcc-10`, although `gcc-9` is still the default.

Reply:
> No, we're not.

I know. That's why I suggested we should. If you'd rather rely on GH installing, that's fine. I was just suggesting being explicit, since we're explicitly using a non-default tool chain.

Reply:
GH is currently including gcc-10 in their list of additional packages they install on top of the "base Ubuntu image + build-essential" image.  I prefer to rely on the latter and explicitly list additional dependencies from that, rather than using GH's custom image as the base.  It's my distro packager OCD shining through.

Reply:
> But I don't see how asking to install something that is already installed (so it's at best a noop) is an improvement?

I think it's just to be explicit to future developers that `gcc-10` is a dependency; this doesn't have any practical effects in and of itself. In the event that we, for example, change CI providers that doesn't have `gcc-10` by default then we'd be sure this still works if we copy paste the steps. It's a "correctness" thing rather than "useful" thing ;)

I'd say the PR is fine as is. If you wanna drop a `gcc-10` in the install list that's fine by me, and if not the let's merge this already :)

Reply:
I guess my point is that we're already explicit by building with `CC=gcc-10`, so I honestly don't see what extra the noop `apt install gcc-10` provides -- when would we expect that to make a practical difference?

If we don't trust availability on the image (which I consider a "custom distro") and have to explicitly install something, we might as well go with the newer `gcc-11` (i.e., drop the second commit).

(I don't care either way; I'm just trying to understand the rationale here.)